cfpb/consumerfinance.gov
# Make pagination Previous/Next and remove customization option

## Changes

- Make pagination Previous/Next and remove customization option


## How to test this PR

1. These pages should have a pagination that says Previous/Next

https://www.consumerfinance.gov/data-research/research-hub/
https://www.consumerfinance.gov/about-us/blog/
https://www.consumerfinance.gov/about-us/events/archive-past-events/
https://www.consumerfinance.gov/about-us/newsroom/
https://www.consumerfinance.gov/activity-log/
https://www.consumerfinance.gov/administrative-adjudication-proceedings/administrative-adjudication-docket/
https://www.consumerfinance.gov/compliance/amicus/briefs/
https://www.consumerfinance.gov/compliance/supervisory-guidance/
https://www.consumerfinance.gov/consumer-tools/auto-loans/answers/ (and all similar Answer results pages across money topic pages)
https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/teach/activities/
https://www.consumerfinance.gov/data-research/cfpb-researchers/
https://www.consumerfinance.gov/data-research/prepaid-accounts/search-agreements/
https://www.consumerfinance.gov/data-research/research-hub/
https://www.consumerfinance.gov/data-research/research-reports/
https://www.consumerfinance.gov/enforcement/actions/
https://www.consumerfinance.gov/enforcement/petitions/
https://www.consumerfinance.gov/es/herramientas-del-consumidor/cobro-de-deudas/respuestas/ (and all similar Respuestas results pages across money topic pages)
https://www.consumerfinance.gov/foia-requests/foia-electronic-reading-room/
https://www.consumerfinance.gov/rules-policy/final-rules/
https://www.consumerfinance.gov/rules-policy/notice-opportunities-comment/archive-closed/
https://www.consumerfinance.gov/rules-policy/rules-under-development/
